packages/luna/src/components/Chat/ChatRoom/ChatRoom.tsx

```diff
@@ -68,70 +86,89 @@ export const ChatRoom: React.FC<{
         }
       )}
     >
-      <div className="tw-w-14 tw-h-14 tw-rounded-full tw-overflow-hidden">
-        <Avatar src={image} alt={name} seed={userId.toString()} />
+      <div className="tw-w-14 tw-h-14 tw-relative tw-rounded-full tw-shrink-0">
+        <div className="tw-overflow-hidden tw-rounded-full tw-relative tw-w-full tw-h-full">
+          <Avatar src={image} alt={name} seed={userId.toString()} />
+        </div>
+        {online && !isActive ? (
+          <div className="tw-absolute tw-bottom-0 tw-right-1 tw-z-activityIndicator tw-w-3 tw-h-3 tw-rounded-full tw-bg-brand-500 tw-border-[.5px] tw-border-natural-50 " />
```

Comment:
I am not sure if using self closing `div` is a good thing. It's not a valid HTML. Despite it's okay with JSX, we may search if it has side effects in compatibility.

Reply:
It doesn't have any side effects, the react transpiler translates this code without any problems.
It's best practice in jsx to use self closing tags in elements with no children, to not crowd the file